omas/omas_physics.py

```diff
@@ -203,6 +228,48 @@ def core_profiles_zeff(ods, update=True, use_electrons_density=False):
                 prof1d_z['zeff'] = Z2n/Zn
     return ods_z
 
+@add_to__ODS__
+def current_from_eq(ods, time_index, set_as='j_ohmic'):
```

Comment:
@bclyons12 I am not sure I fully understand the purpose of this function. It looks like you are trying to take the total current from an equilibrium and set it as either the ohmic, bootstrap, or noninductive components in the core_profiles IDS. Shoudn't we set the total current density in the equilibrium IDS equal to the total current density in the core_profiles IDS, and that's it?

Reply:
@orso82 Let's discuss in person. The way the `update_current` function works, if you don't set the components of the current, it'll behave strangely.

Reply:
@orso82 We can merge #48, then I'll update `current_from_eq` here and we can merge that one.

Reply:
@bclyons12 I'll merge `update_current` in `core_profiles_consistent` so that it can all be tested here. I'll also work on setting up a regression test for the `update_current()` function with the hope of inspiring you to do the same for `current_from_eq()` \*wink\* \*wink\*  ;)

Reply:
@orso82 sounds good